theme/tokens.ts

```diff
@@ -1,7 +1,23 @@
 import { tokens as leatherTokens } from '@leather-wallet/tokens';
 import { defineTokens } from '@pandacss/dev';
 
+import { colors } from './colors';
+
 // ts-unused-exports:disable-next-line
 export const tokens = defineTokens({
   ...leatherTokens,
+  // TODO: Update in mono repo
```

Comment:
Maybe we can update the `monorepo` now it's not such a pain and test the new process again? 

You will need to update the extension to be `0.0.8` so may as well be `0.0.9`